### DISCUSSION TITLE
New Developer / Webbased App with Sound Output

### DISCUSSION THREAD

**Discussion by ludwich66** [webspiderteam/Bluetooth-DMM-For-Windows#41] [General]
Hello Engine,
there is a new developer for the BT-DMM, here only your device. You can see if it works. 

https://github.com/bendtherules/multimeter-connect-web
https://multimeter-speaker.netlify.app

Greetings Ludwich

**Comment by webspiderteam** [comment]
Nice usage :) has some problems but its working :) and congragulations to you :) its added yesterday :) but looks like he didn't mentioned my app because i already encoded all of protocols but he decoded only one :)

**Reply by webspiderteam** [reply]
btw looks like its only working on smart phones :) not on windows

**Reply by ludwich66** [reply]
He left a message that the diode (BigDMM, Pos56) is missing in my Wiki, can you please check that again with your documents?

https://github.com/ludwich66/Bluetooth-DMM/wiki/Protocol-all-Variants

"In wiki page (https://github.com/ludwich66/Bluetooth-DMM/wiki/Protocol-all-Variants), Diode: 56 is missing for Big-DMM type. Please add it."

**Reply by webspiderteam** [reply]
yes its in 56 
![image](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/assets/85828505/e2a75210-dbb2-4b64-9289-0dce0f109cdc)

**Reply by webspiderteam** [reply]
some how it looks like slipped some where. do you want me to fix them?

**Comment by webspiderteam** [comment]
I made the correction on your wiki ;) BTW https://tableconvert.com/ helped me alot its more easy while creating table you should try :)

**Reply by ludwich66** [reply]
Thanks alot!!

**Comment by ludwich66** [comment]
Can you code at this temperatures?
How warm is it?
Munich Yesterday 36° Today 29°

**Comment by webspiderteam** [comment]
ofcourse I can :) it was 40 yesterday today was 37
